freeipa.spec.in

```diff
@@ -192,7 +192,7 @@ BuildRequires:  python2-dns >= 1.15
 BuildRequires:  jsl
 BuildRequires:  python2-yubico
 # pki Python package
-BuildRequires:  pki-base-python2
+BuildRequires:  pki-base-python2 >= 10.5.1-2
```

Comment:
Do you know of plans to push this build to F27/rawhide?

Reply:
No, somebody has to talk to Matt. @mharmsen99?

Reply:
Matt told us probably next week so that's fine. ACK.